src/app/app.tsx

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import { StoreContext } from 'storeon/react';
+import './app.scss';
+import { AboutPage } from './components/aboutPage/aboutPage.component';
 import { AppStore } from './app.store';
-import { Router, View } from 'react-navi';
-import { compose, mount, route, withView } from 'navi';
 import { Articles } from './components/articles/articles.component';
-import { Reports } from './components/reports/reports.component';
-import { MainHeader } from './components/header/header.component';
+import { Blogs } from './components/blogs/blogs.component';
+import { compose, mount, route, withView } from 'navi';
 import { Footer } from './components/footer/footer.component';
 import { MainContent } from './components/mainContent/mainContent.component';
-import { Blogs } from './components/blogs/blogs.component';
-import { AboutPage } from './components/aboutPage/aboutPage.component';
-import './app.scss';
+import { MainHeader } from './components/header/header.component';
+import { Reports } from './components/reports/reports.component';
+import { Router, View } from 'react-navi';
+import { StoreContext } from 'storeon/react';
+import React from 'react';
```

Comment:
React import should be at the very top. Please rearrange imports to be in logical order

Reply:
Imports sorted using standard convention - first import of the React package, then imports of the third-party libraries and in the end imports of the local files. All grouped separately. Applied to all tsx files.